src/subapps/dataExplorer/DataExplorerTable.tsx

```diff
@@ -353,7 +352,7 @@ const initialTableConfig = (showEmptyDataCells: boolean) => {
 
   const projectConfig: ColumnType<Resource> = {
     ...defaultColumnConfig(projectKey, showEmptyDataCells),
-    title: 'PROJECT',
+    title: '_project',
```

Comment:
Nice to have: Can we use `projectKey` and `typeKey` constants here for consistency?